lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_localize_helper.rb

```diff
@@ -11,19 +11,19 @@ module Android
       module LocalizeHelper
         LIB_SOURCE_XML_ATTR = 'a8c-src-lib'.freeze
 
-        # Checks if string_line has the content_override flag set
-        def self.skip_string_by_tag(string_line)
-          skip = string_line.attr('content_override') == 'true' unless string_line.attr('content_override').nil?
+        # Checks if `string_node` has the content_override flag set
```

Comment:
Nitpick, given one is surrounded in backticks, we might want to do the same for the other:

```suggestion
        # Checks if `string_node` has the `content_override` flag set
```

Reply:
Missed that before merging, but still included the suggested change in the #359 via fb72adc before doing the release 😅